helm/prometheus-rules/templates/alerting-rules/prometheus-agent.rules.yml

```diff
@@ -17,6 +17,7 @@ spec:
         description: '{{`Prometheus agent remote write is failing.`}}'
         summary: Prometheus agent fails to send samples to remote write endpoint.
         opsrecipe: prometheus-agent-remote-write-failed/
+        dashboard: BG-1wO5Vk/prometheus-remote-write
```

Comment:
Is the URL the same between each MC ?

Reply:
The dashboard UID and the dashboard name are the same, yes.
Alertmanager will build the address pointing to the installation's grafana.

Reply:
Argh, you're right:
Most dashboards have a `uid` defined (see https://github.com/giantswarm/dashboards/blob/master/helm/dashboards/dashboards/shared/private/credentials.json#L419), but some don't.
For instance, prometheus-remote-write does not have it defined.
So, grafana generates a random one.

So, I'll have to:
- update dashboard to have a `uid`
- update `prometheus-rules` documentation with this info